images/runtime/python/template.Dockerfile

```diff
@@ -77,8 +77,9 @@ RUN ${IMAGES_DIR}/runtime/python/install-dependencies.sh
 RUN pip install --upgrade pip \
     && pip install gunicorn \
     && pip install debugpy \
-    && if [ "%PYTHON_MAJOR_VERSION%" = "3" ] && [ "%PYTHON_VERSION%" != "3.6" ]; then pip install viztracer==0.14.3 \
-    && pip install vizplugins==0.1.2 \
+    && if [ "%PYTHON_MAJOR_VERSION%" = "3" ] && [ "%PYTHON_VERSION%" != "3.6" ] && [ "%PYTHON_VERSION%" != "3.11" ]; then pip install viztracer==0.14.3; fi\
```

Comment:
Investigating this a little more -- from the change logs for the releases between `0.14.3` and `0.15.5` in the `viztracer` repository, it doesn't appear that anything notable has changed other than dropped support for Python 3.6, but that version has reached end-of-life and Antares doesn't consume it anymore. I'd be OK with us adopting `0.15.5` for all versions of Python that we publish now, letting Antares know about this bump and having them test out any `viztracer` functionality they expect to be supported with their Python app tests. Worst case scenario, they can still use the new Python 3.11 runtime (since `0.15.5` is the only version support for Python 3.11), and we can bring back this conditional to install `0.14.3` for Python 3.7 to 3.10.

@daniv-msft -- any thoughts on this? (more context in the description of the PR)